kinode/src/main.rs

```diff
@@ -377,6 +387,24 @@ async fn main() {
         }
     };
 
+    // read in default routers .json file
+    type KnsUpdate = crate::net::KnsUpdate;
+    let routers: Vec<KnsUpdate> =
+        match fs::read_to_string(format!("{}/.routers", home_directory_path)).await {
+            Ok(contents) => serde_json::from_str(&contents).unwrap(),
+            Err(_) => {
+                let routers: Vec<KnsUpdate> = serde_json::from_str(DEFAULT_ROUTERS).unwrap();
+                // should we dump them into a .routers file?
+                // fs::write(
+                //     format!("{}/.routers", home_directory_path),
+                //     serde_json::to_string(&routers).unwrap(),
+                // )
+                // .await
+                // .unwrap();
+                routers
+            }
```

Comment:
also left this commented out as a question, probably going to remove it, letting user specifically write .routers if necessary. perhaps should have a specific eth-rpc node config too?